docs/integrations/javascript/browser.md

```diff
@@ -0,0 +1,40 @@
+---
+integration: logfire
+---
+
+# Browser
```

Comment:
not sure, but might be helpful to mention if bundler configuration is needed to support ESM

Reply:
And also maybe we can add minimum version requirements of Node, Deno, etc., if applicable.